caluma/caluma_workflow/migrations/0020_case_families.py

```diff
@@ -0,0 +1,60 @@
+# Generated by Django 2.2.10 on 2020-03-05 09:17
+
+import django.db.models.deletion
+from django.db import migrations, models
+
+
+def set_family(apps, schema_editor):
+    Case = apps.get_model("caluma_workflow", "Case")
+    HistoricalCase = apps.get_model("caluma_workflow", "HistoricalCase")
+    db_alias = schema_editor.connection.alias
+
+    orphan_cases = Case.objects.using(db_alias).filter(family__isnull=True)
+    orphan_historical_cases = HistoricalCase.objects.using(db_alias).filter(
+        family__isnull=True
+    )
+
+    for case in orphan_cases:
+        case.family = case
```

Comment:
This doesn't set the case's family correctly IMHO, as it doesn't take the family structure into account. I think we should actually do that, so even "old" cases will have the correct structure.

Reply:
You're absolutely right. It's fixed. Historical cases referencing non-existent cases, are treated as their own family.